tests/unittests/tensorsTest.cpp

```diff
@@ -146,7 +146,7 @@ TEST(Tensor, assignment) {
   EXPECT_EQ(Handle.at({10, 10, 10}), 10 + 10 + 10);
 
   auto TT = Handle.extractSlice(1);
-  auto H2 = TT.getHandle<>();
+  auto H2 = TT.template getHandle<Ty>();
```

Comment:
probably "template" is not required here?

Reply:
I removed "template" by declaring `TT` using a specific type `Tensor`.